scene/main/viewport.cpp

```diff
@@ -738,6 +738,14 @@ void Viewport::_process_picking() {
 
 	while (physics_picking_events.size()) {
 		local_input_handled = false;
+		if (!handle_input_locally) {
+			Viewport *vp = this;
+			while (!Object::cast_to<Window>(vp) && vp->get_parent()) {
+				vp = vp->get_parent()->get_viewport();
+			}
+			vp->local_input_handled = false;
+		}
+
```

Comment:
Just to be sure, you seem to skip every viewport (even ones with `local_input_handled == true` until the last one, then you set `local_input_handled` to `false`.
Is this what you want?

Reply:
This PR intends to treat `handle_input_locally` for physics picking in the same way as it is treated in other places. That includes skipping viewports with `local_input_handled == true`.
The fact that currently `handle_input_locally` is treated differently in different parts of the code-base is the reason for the linked issue.